helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -95,8 +95,12 @@ def _compute_duplicate_count(self):
     partner_name = fields.Char()
     partner_email = fields.Char(string="Email")
     last_stage_update = fields.Datetime(default=fields.Datetime.now)
-    assigned_date = fields.Datetime()
-    closed_date = fields.Datetime()
+    assigned_date = fields.Datetime(
+        copy=False,
+    )
+    closed_date = fields.Datetime(
+        copy=False,
+    )
```

Comment:
The change is OK, but please don't put trailing comma and waste lines of code.

```suggestion
    assigned_date = fields.Datetime(copy=False)
    closed_date = fields.Datetime(copy=False)
```